aws/v12.3.0/README.md

```diff
@@ -0,0 +1,29 @@
+## :zap:  Giant Swarm Release 12.3.0 for AWS :zap:
+
+**If you are upgrading from 12.2.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades all Helm releases managed by Giant Swarm to use [Helm v3.3.4](https://github.com/helm/helm/releases/tag/v3.3.4).
+
+This lets us benefit from the improved security model and keep up to date with the community. We also remove the Tiller deployment from the giantswarm namespace, removing its gRPC endpoint, which reduces operational complexity.
+
+If you are still using Helm 2 then these Helm releases will not be affected. However we encourage you to upgrade to Helm 3. As Helm 2 support ends on November 13th 2020. https://helm.sh/blog/helm-v2-deprecation-timeline/
+
+Below, you can find more details on components that were changed with this release.
+
+**Note before upgrade:**
+
+Please contact your Solution Engineer before upgrading. The upgrade is automated. However, it includes a data migration from Helm 2 release configmaps to Helm 3 release secrets, there are some pre-upgrade checks and we recommend monitoring the upgrade to ensure safety.
+
+**Note for Solution Engineers:**
```

Comment:
When we did the preview release, we discovered the known issue about `deployed` vs `DEPLOYED` causing problems w MS Teams notification. Should we add that as a section here https://intranet.giantswarm.io/docs/dev-and-releng/helm/helm3-tenant-cluster-upgrade/?

Reply:
Good idea, I will add that information into our doc.